parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -41,7 +41,7 @@ installDeps() {
     packages+=" cgroup-lite ceph-common glusterfs-client"
     if [[ $UBUNTU_RELEASE == "18.04" ]]; then
       disableTimeSyncd
```

Comment:
Given that azsecpack enables timesyncd anyways should we just leave the timesyncd config as-is from the Canonical image, and delete the `disableTimeSyncd` foo altogether?

Reply:
azsecpack is going to remove that logic (or only conditionally enable timesyncd when chrony is NOT there).

Disabling timesyncd is a must prerequisite for running chrony correctly, so we should keep disabling timesyncd here (aks-engine should not worry about azsecpack).